guides/v2.3/graphql/queries/products-234.md

```diff
@@ -0,0 +1,1144 @@
+---
+group: graphql
+title: products query (Magento 2.3.4 and later)
+---
+
+The `products` query allows you to search for catalog items.
+
+## Syntax
+
+```graphql
+products(
+  search: String
+  filter: ProductAttributeFilterInput
+  pageSize: Int
+  currentPage: Int
+  sort: ProductSortFilterInput
+): Products
+```
+
+## Input attributes
+
+Each query attribute is defined below:
+
+Attribute |  Data type | Description
+--- | --- | ---
+`search` | String | Performs a full-text search using the specified key words. This attribute is optional. S
+`filter` | ProductAttributeFilterInput | Identifies which attributes to search for and return. This attribute is required. See [filter attribute](#filter) object for more information.
+`pageSize` | Int | Specifies the maximum number of results to return at once. The default value is 20.
+`currentPage` | Int | Specifies which page of results to return. The default value is 1.
+`sort` | ProductAttributeSortInput | Specifies which attribute to sort on, and whether to return the results in ascending or descending order.
+
+### search attribute
+
+The `search` element causes Magento to perform a full text search on the specified keywords. (This is the same type of search that is performed from the storefront. If multiple keywords are specified, each keyword is evaluated separately.)
```

Comment:
I don't think the parenthetical phrase needs to be in parentheses. It's either a full sentence as is or a note.